packages/gamepads_android/README.md

```diff
@@ -0,0 +1 @@
+../../README.md
```

Comment:
Do the other packages also have symlinks?
Since you can't publish a package with symlinks from most platforms, but we might have just removed the symlink before publishing.

Reply:
Yeah

https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_darwin/README.md
https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_ios/README.md
https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_linux/README.md
https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_windows/README.md